src/components/Pop-Up-Modal/EditDescription.vue

```diff
@@ -36,13 +36,17 @@
           placeholder="Description Item"
           class="typeInput"
         />
-        <p>Unit : </p>
-        <input
-          type="text"
-          v-model="processedData.description_unit"
-          placeholder="Unit"
-          class="typeInput"
-        />
+
+        <div v-if="processedData.description_unit">
```

Comment:
can edit from BODY to HEAD by making the unit blank? Suggest better to prevent it also

Reply:
Okay, then I already re-update again the code, so when the unit is blank, I will give it to save ' - ', then make sure the body description unit is not empty, just '- ' and have data.